.cirrus.yml

```diff
@@ -14,12 +14,6 @@ zkg_task:
   container:
     dockerfile: ci/Dockerfile
     matrix:
-      - docker_arguments:
-        - ZEEK_VERSION: "v4.0.3"
-        - ZEEK_CONFFLAGS: ""
-      - docker_arguments:
-        - ZEEK_VERSION: "v4.2.1"
-        - ZEEK_CONFFLAGS: ""
```

Comment:
Let's add the most recent 5.x version.

Reply:
Not sure I am missing something, but just below we run against `v5.1.1` which should be the latest release.

Reply:
I'm just blind. :-)